vmdk_plugin/plugin.go

```diff
@@ -180,7 +180,7 @@ loop:
 		case <-time.After(devWaitTimeout):
 			log.WithFields(
 				log.Fields{"name": name, "timeout": devWaitTimeout, "device": device},
-			).Warning("Reached timeout while waiting for device, trying to mount anyways ")
```

Comment:
Sorry thats a merge issue. Will revert.

On Tue, Aug 2, 2016 at 10:39 AM, Prashant Dhamdhere <
notifications@github.com> wrote:

> In vmdk_plugin/plugin.go
> <https://github.com/vmware/docker-volume-vsphere/pull/566#discussion_r73094923>
> :
>
> > @@ -180,7 +180,7 @@ loop:
> >  		case <-time.After(devWaitTimeout):
> >  			log.WithFields(
> >  				log.Fields{"name": name, "timeout": devWaitTimeout, "device": device},
> > -			).Warning("Reached timeout while waiting for device, trying to mount anyways ")
>
> Warning is correct because we still proceed and attempt to mount.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/566/files/d7be9d7eed212200b348ef3c9a49207af529eb0e#r73094923>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APHseMsZhuUHwCnzx72xkzamoHlisg_Uks5qbtD9gaJpZM4JaPm_>
> .
>